utils/ccloud_library.sh

```diff
@@ -932,6 +944,25 @@ function ccloud::set_kafka_cluster_use() {
 }
 
 
+function ccloud::maybe_create_and_use_environment() {
+
+  ENVIORNMENT_NAME=$1
```

Comment:
```suggestion
  ENVIRONMENT_NAME=$1
```

Reply:
Thanks for catching this!  It was actually misspelled in TWO places, which is why it worked when I tested it.  Fixed in latest commit.